bin/cyhy-import

```diff
@@ -79,6 +80,16 @@ def has_intersections(db, nets, filename, owner):
         print_intersections(intersections)
         return True
 
+def has_restricted_ips(nets):
+    geo_loc_db = GeoLocDB()
+    has_restricted = False
+    for cidr in nets.iter_cidrs():
+        if geo_loc_db.check_restricted_cidr(cidr):
+            has_restricted = True
+    if has_restricted:
+        return True
+    else:
+        return False
```

Comment:
This logic can be simplified considerably:
```suggestion
    for cidr in nets.iter_cidrs():
        if geo_loc_db.check_restricted_cidr(cidr):
            return True
    return False
```

Reply:
The reason I had it like this was so it had the chance to iterate through all the cidrs before returning, in case there were multiples so the user would not have to keep going back. Is there an easier way to do this?

Reply:
This would simplify while still allowing it to iterate through all of the CIDRs:

```suggestion
    has_restricted = False
    for cidr in nets.iter_cidrs():
        if geo_loc_db.check_restricted_cidr(cidr):
            has_restricted = True

    return has_restricted
```

Reply:
See [here](https://github.com/cisagov/cyhy-core/pull/60#discussion_r595360456) for my response.